executable_schema.go

```diff
@@ -155,6 +180,21 @@ func (s *ExecutableSchema) ExecuteQuery(ctx context.Context) *graphql.Response {
 	operation := operationCtx.Operation
 	variables := operationCtx.Variables
 
+	ctx, span := s.tracer.Start(ctx, "Executable Schema Query",
+		trace.WithSpanKind(trace.SpanKindInternal),
+	)
+
+	defer span.End()
+
+	handleErr := func(err error) {
```

Comment:
perhaps give this a less generic name and say what it is doing? `traceError`, `recordError` or `addErrorToSpan`